extensions/sockets/src/test/java/org/mule/extension/socket/SocketListenerReconnectionTestCase.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.socket;
+
+public class SocketListenerReconnectionTestCase
```

Comment:
I have some questions for the team about this, but we can omit this test on this PR and treat it in https://www.mulesoft.org/jira/browse/MULE-9680